std_msgs/Header header

uint32 t_us
uint16 seq
uint8 flags

float32 roll
float32 pitch
float32 yaw

float32 W_e_acc
float32 W_beta

float32 A_e_acc
float32 A_beta

uint8 fusion_mode   # 0=6axis, 1=9axis